keyboards/xd84/xd84.c

```diff
@@ -15,29 +15,18 @@
  */
 #include "xd84.h"
 
-void matrix_init_kb(void) {
-	// put your keyboard start-up code here
-	// runs once when the firmware starts up
+void keyboard_pre_init_kb(void) {
+  setPinOutput(B6);
```

Comment:
Perfect! 

i'm glad to see this used this way!

Reply:
Its really tidied up quite a few scenarios, especially the hacks in my userspace.